### PR TITLE
Material updates to include Tronxy machines

### PR DESCRIPTION
**Pull request by 64bittuning**
#129 was accidentally deleted from my repositories
Re-opening PR

### CODE REVIEW

**Comment by Ghostkeeper** [comment]
Developers, merge this at the same time as https://github.com/Ultimaker/Cura/pull/7622.

**Comment by Ghostkeeper** [comment]
Oops. Thanks for pinging...

File: generic_petg_175.xml.fdm_material [hunk 1]
```diff
@@ -59,11 +59,28 @@ Generic PETG 1.75mm profile. The data in this file may not be correct for your s
...
             <machine_identifier manufacturer="Creality3D" product="creality_ender4" />
             <machine_identifier manufacturer="Creality3D" product="creality_ender5" />
-            <setting key="print cooling">50</setting>
+            <setting key="print cooling">100</setting>
             <setting key="standby temperature">200</setting>
-            <setting key="retraction speed">45</setting>
+            <setting key="retraction speed">35</setting>
             <setting key="retraction amount">5</setting>
             <setting key="print temperature">240</setting>
+            <setting key="heated bed temperature">80</setting>
```
Comment (Ghostkeeper) [inline generic_petg_175.xml.fdm_material:67]: This changes settings for the Creality printers. I don't think you intended to do that.

Comment (64bittuning) [reply generic_petg_175.xml.fdm_material:67]: I didn't mean to ping the PR. I hit the wrong button in the GitHub app while poking around. Totally an accident.

If I made changes to the Creality Profiles, it was unintentional.

Comment (Ghostkeeper) [reply generic_petg_175.xml.fdm_material:67]: I've reverted the changes to the Creality profiles but merged your other changes. They should've been merged together with your PR to Cura, but we forgot.

**Pull request merged**